_posts/2016-04-05-extension-framework-game-plan.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+cover: 'assets/images/general-cover-3.jpg'
+title: Extension Framework Game Plan
+tags: docs
+subclass: 'post tag-docs'
+categories: 'elixir'
+author: 'Pikender'
+navigation: true
+logo: 'assets/images/nectar-cart.png'
+---
+
+### What will be NectarCommerce
+
+>
+Off-the-shelf Opensource E-commerce application for building online store.
+>
+Provides an Extension Framework to support features not included in core as extensions.
+>
+Strives for un-obstrusive parallel development of NectarCommerce and Extensions
+
+NectarCommerce is committed to provide a ready-to-use e-commerce solution but definition of 100% is different under different business domains.
+
+NectarCommerce aims to solve trivial use-cases as part of the project and relying on extension framework to tap the rest.
+
+## Extension Framework Game Plan
+
+Let's validate and list the capabilities needed in Extension Framework through an **extension** which provides feature **to favorite products by Users.**
+
+_**Favorite Product Extension Requirements:**_
+
+- **Ability to mark and unmark a Product as favorite**
+  - a View _probably_ showing all products with Abilityity to mark/unmark product as favorite
```

Comment:
Fixed typo.

probably is distinguished with italics to invoke user thought. I can add brackets, no issues there.

@chiragmongia Let me know what you think